app/src/lib/ui/Manager/ImportBeneficiaries.svelte

```diff
@@ -147,18 +162,18 @@
 	async function handleFilesSelect(event: CustomEvent<{ acceptedFiles: Buffer[] }>): Promise<void> {
 		parseErrors = [];
 		files = event.detail.acceptedFiles;
+		parsing = true;
 		for (let i = 0; i < files.length; i++) {
-			const parsingResponse = await fileValidationApi(files[i]);
-			const responseBody = await parsingResponse.json();
-			if (parsingResponse.ok) {
-				beneficiaries = responseBody.map((line) => beneficiaryFrom(line));
+			try {
```

Comment:
pour plus de clarté, je propose d'extraire le contenu du `for` dans une fonction